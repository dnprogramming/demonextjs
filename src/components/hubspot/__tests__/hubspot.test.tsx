import { render } from "@testing-library/react";
import Hubspot from "../hubspot";
import React from "react";

describe("Hubspot Component", () => {
    it("renders Hubspot component unchanged", () => {
        const { container } = render(<Hubspot />);
        expect(container).toMatchSnapshot();
    });
});