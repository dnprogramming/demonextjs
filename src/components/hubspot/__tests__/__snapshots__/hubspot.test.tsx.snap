// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Hubspot Component renders Hubspot component unchanged 1`] = `<div />`;
